UnityProject/Assets/Scripts/Items/Food/DrinkableContainer.cs

```diff
@@ -79,11 +81,27 @@ public override void TryConsume(GameObject feederGO, GameObject eaterGO)
 	private void Drink(PlayerScript eater, PlayerScript feeder)
 	{
 		// Start drinking reagent mix
-		// todo: actually transfer reagent mix inside player stomach
 		var drinkAmount = container.TransferAmount;
-		container.TakeReagents(drinkAmount);
-
-		DoDrinkEffects(eater, drinkAmount);
+		int stomachIndex = 0;
+		List<Stomach> stomachs = eater.playerHealth.GetStomachs();
+		while (drinkAmount > 0)
+		{
+			ReagentContainer stomachContainer = stomachs[stomachIndex].StomachContents;
+			if (stomachContainer.IsFull)
+			{
+				if (stomachIndex < stomachs.Count-1)
```

Comment:
what does this do?

Reply:
Transfers the reagents in the DrinkableContainer to each stomach as the preceding stomach fills up until the drink is empty. Currently rewriting this on local.